src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1244,7 +1244,6 @@ export default class DefaultDeviceController
   }
 
   private async restartLocalVideoAfterSelection(
-    oldDevice: DeviceSelection,
```

Comment:
Just verifying if this is updated at all places where called.

Reply:
yup - global search returned nothing.